src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -492,4 +492,25 @@
   <data name="SuppressDiagnosticWithCommentNonNestedCodeActionTitle" xml:space="preserve">
     <value>Suppress the {0} diagnostic with Acuminator in a comment</value>
   </data>
+  <data name="PX1007Class" xml:space="preserve">
+    <value>class</value>
+  </data>
+  <data name="PX1007Delegate" xml:space="preserve">
+    <value>delegate</value>
+  </data>
+  <data name="PX1007Enum" xml:space="preserve">
+    <value>enum</value>
```

Comment:
enumeration

Reply:
@k-s-popova are you sure? The enum is pretty wide-known term

Reply:
Yes. We have enumeration in our API reference, and Microsoft uses enumeration in the description of the enum keyword: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/enum

Reply:
ok, I'll fix it

Reply:
done